src/room/LobbyView.module.css

```diff
@@ -24,6 +24,10 @@ limitations under the License.
   overflow: hidden;
   height: 100%;
 }
+.wait {
+  color: black !important;
+  background-color: rgba(255, 255, 255, 0.67) !important;
+}
```

Comment:
I'm suspicious of overriding our component colors like this, especially when the colors aren't theme-reactive. Could we use a standard button state that has already been designed, like a disabled state?

Reply:
I played around with the standard button state and did not like it. I forgot the exact issues however.

Reply:
The reason for this was, that the disabled button is the wrong color (dark in dark theme and light in light theme)

Reply:
This really should use compound color. I need transparent backgrounds however. I tried relative colors but this was not supported on firefox...